Hubitat Virtual Camera Object driver.groovy

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright 2020 Lyle Pakula
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Camera Stream Object
+ *
+ *  Drive does nothing but bridge an HTTP stream into Google Home to display as a video camera.
+ *  Saving the preferences creates the video URL link: http://${deviceIP}/h264/${deviceName}/temp.m3u8?user=${deviceUser}&pw=${devicePWD} which will be passed to a Chromecast device
```

Comment:
This seems pretty specific to a particular model of camera.  Maybe it would make more sense for this to be a standalone driver?  Or maybe make it more generic somehow?

Reply:
Let me split this into two drivers.  A generic where the user can just enter the specific URL for the stream, and re-label this on which is specific for the BlueIris DVR.